source/includes/k8s/file-transfer-protocol-k8s.rst

```diff
@@ -0,0 +1,167 @@
+.. versionadded:: Operator v5.0.7
```

Comment:
This file is the new `k8s` content specifically for Operator. SFTP only, no other protocols.